Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Property.java

```diff
@@ -31,9 +32,11 @@
 @Table(name = "property")
```

Comment:
Is an `@Indexed(index = "kitodo-property")` annotation not missing like in the other bean files?

Reply:
My understanding is that the annotation only applies to objects to be indexed. Standalone properties are not indexed as separate objects.

Reply:
I understand.